apps/teams-perf-test-app/package.json

```diff
@@ -5,9 +5,9 @@
   "description": "Teams Perf Test App utilizing Teams JavaScript client SDK to test Teams Host Perf",
   "version": "0.0.1",
   "scripts": {
-    "build": "yarn eslint ./src/**/*.tsx && webpack",
+    "build": "yarn lint && webpack",
     "clean": "rimraf ./build",
-    "eslint-fix-build": "yarn eslint --fix --ext .tsx ./src && webpack",
+    "lint": "yarn eslint ./src --max-warnings 0 --fix --ext .tsx",
```

Comment:
>tsx [](http://example.com/codeflow?start=61&length=3)

I know this isn't new (and this question applies to the other package.json files in which it appears) -- but does this cause us to not lint .TS files also?

Reply:
I left it as .tsx for the app package.jsons since all they have are .tsx files and it throws an error saying no files matching the pattern ".ts" were found.

Reply:
does the script accept a conditional like `(ts|tsx)$`

Reply:
Hmm I think it might be possible, I tested a few ways to use it and that didn't work though. (`(.ts|.tsx)`, `(.ts|.tsx)$`, `$(.ts|.tsx)`)

Reply:
Seems crazy that you can't allow for future expansion into a different file extension without causing the error you say but 🤷‍♂️

Does .ts* work?  :)

Reply:
I tried variations using those ( moving the argument order around, etc.) but I can't seem to make it work. Maybe I can open a bug item so we're prepared for any .ts files coming in?

Reply:
I don't think I would open a bug for it since it's not really actionable now and I don't think it's going to cause anyone who happens to add a TS file to remember to do something. I guess we just have to leave it like this for now.

Reply:
I was thinking maybe they could figure out how to add the .ts extension without there being ts files in the mo 😆